WORKSPACE

```diff
@@ -27,3 +27,8 @@ boost_deps()
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
 grpc_deps()
+
+load("@prometheus_cpp//:repositories.bzl", "load_prometheus_client_model", "load_civetweb")
+
+load_prometheus_client_model()
```

Comment:
Please run buildifier on this file. I think it will add an empty line between the two load_... calls.

Reply:
Done, thanks!